express-notes-api/index.js

```diff
@@ -0,0 +1,106 @@
+const fs = require('fs');
+const express = require('express');
+const app = express();
+const errorMsg = {
+  error: 'id must be a postive integer'
+};
+const errorMsgEmpty = {
+  error: 'content is a required field'
+};
+const errorUnexpected = {
+  error: 'An unexpected error occurred.'
+};
+app.listen(3000, () => {
+});
```

Comment:
You should move this into your readFile callback so that the server doesn't start handling requests until all of its routes are set up.